hadoop-ozone/s3gateway/src/test/java/org/apache/hadoop/ozone/s3/endpoint/TestPartUpload.java

```diff
@@ -135,4 +141,70 @@ public void testPartUploadWithIncorrectUploadID() throws Exception {
       assertEquals(HTTP_NOT_FOUND, ex.getHttpCode());
     }
   }
+
+  @Test
+  public void testPartUploadStreamContentLength()
+      throws IOException, OS3Exception {
+    HttpHeaders headers = Mockito.mock(HttpHeaders.class);
+    ObjectEndpoint objectEndpoint = new ObjectEndpoint();
+    objectEndpoint.setHeaders(headers);
+    objectEndpoint.setClient(client);
+    objectEndpoint.setOzoneConfiguration(new OzoneConfiguration());
+    objectEndpoint.setHeaders(headers);
+    String keyName = UUID.randomUUID().toString();
+
+    String chunkedContent = "0a;chunk-signature=signature\r\n"
```

Comment:
@xichen01 , could you explain a little bit about this chunkedContent? Don't understand why this string length is 15.

Reply:
Okay.